client/src/www/app/app.ts

```diff
@@ -880,8 +868,27 @@ export class App {
     return !('cordova' in window);
   }
 
-  private setAppTheme(event: CustomEvent) {
-    const theme = event.detail.themePreference;
-    this.themeManager.setThemePreference(theme as ThemePreference);
+  private setAppearance(appearance: Appearance) {
+    const documentClassList = window.document.documentElement.classList;
+    const isSystemDark = matchMedia('(prefers-color-scheme: dark)').matches;
+
+    let applyDarkTheme;
+
+    if (appearance === Appearance.DARK) {
+      applyDarkTheme = true;
+    } else if (appearance === Appearance.LIGHT) {
+      applyDarkTheme = false;
+    } else {
+      // Theme.SYSTEM or any other default
+      applyDarkTheme = isSystemDark;
```

Comment:
Just ran the code a second time with the feature flag off. The app automatically went dark. The expected behaviour is to only trigger a check for dark, light or system if the feature flag is on. (Of course if you are going to merge with the flag on - then this wouldn't matter.)